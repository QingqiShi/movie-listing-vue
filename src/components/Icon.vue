<template>
  <i class="material-icons">{{ name }}</i>
</template>

<script>
export default {
  props: {
    name: { type: String, default: "home" }
  }
};
</script>
