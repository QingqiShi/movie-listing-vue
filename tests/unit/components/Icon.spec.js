import { shallowMount } from "@vue/test-utils";
import Icon from "@/components/Icon.vue";

describe("Icon.vue", () => {
  it("renders", () => {
    const wrapper = shallowMount(Icon, { propsData: { name: "test" } });
    expect(wrapper).toMatchSnapshot();
  });
});
