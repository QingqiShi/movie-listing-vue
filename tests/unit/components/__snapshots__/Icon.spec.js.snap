// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon.vue renders 1`] = `<i class="material-icons">test</i>`;
